apps/core/app/[locale]/(default)/account/[tab]/_components/addresses-content.tsx

```diff
@@ -0,0 +1,40 @@
+import { getLocale, getTranslations } from 'next-intl/server';
+
+import { getCustomerAddresses } from '~/client/queries/get-customer-addresses';
+import { LocaleType } from '~/i18n';
+
+import { Pagination } from '../../../(faceted)/_components/pagination';
+import { TabType } from '../layout';
+import { tabHeading } from '../page';
+
+import { AddressesList } from './addresses-list';
+
+type CustomerAddresses = NonNullable<Awaited<ReturnType<typeof getCustomerAddresses>>>;
+
+interface Props {
+  addresses: CustomerAddresses['addresses'];
+  pageInfo: CustomerAddresses['pageInfo'];
+  title: TabType;
+}
+
+export const AddressesContent = async ({ addresses, pageInfo, title }: Props) => {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  const locale = (await getLocale()) as LocaleType;
```

Comment:
```suggestion
  const locale = await getLocale();
```

Probably you don't need type assertion here

Reply:
Yeah, I've simplified type here to `string`.